core/src/components/checkbox/checkbox.tsx

```diff
@@ -98,6 +98,11 @@ export class Checkbox implements ComponentInterface {
    */
   @Prop() alignment: 'start' | 'center' = 'center';
 
+  /**
+   * Set to `"small"` for a checkbox with less height and padding or to `"default"` for a checkbox with the default height and padding.
```

Comment:
Update this comment to remove the comment about the `"default"` option. 

Suggestion:
> Set to `"small"` for a checkbox with less height and padding.